README.md

```diff
@@ -1,44 +1,69 @@
-## scratch-www
+# scratch-www
 #### Standalone web client for Scratch
 
 [![Build Status](https://travis-ci.org/LLK/scratch-www.svg)](https://travis-ci.org/LLK/scratch-www)
 [![Coverage Status](https://coveralls.io/repos/github/LLK/scratch-www/badge.svg?branch=develop)](https://coveralls.io/github/LLK/scratch-www?branch=develop)
-[![dependencies Status](https://david-dm.org/llk/scratch-www/status.svg)](https://david-dm.org/llk/scratch-www)
-[![devDependencies Status](https://david-dm.org/llk/scratch-www/dev-status.svg)](https://david-dm.org/llk/scratch-www?type=dev) [![Greenkeeper badge](https://badges.greenkeeper.io/LLK/scratch-www.svg)](https://greenkeeper.io/)
+[![Greenkeeper badge](https://badges.greenkeeper.io/LLK/scratch-www.svg)](https://greenkeeper.io/)
 
-### Where am I?
-Physically? No idea.
+## Overview
 
-Digitally? You’re at Scratch’s open source web client!
+This is Scratch’s open source web client! This is the code for much of the [Scratch website](https://scratch.mit.edu).
 
-We’re working to update the [Scratch website](https://scratch.mit.edu) to use a new codebase, contained in this repository.
+In particular, this codebase includes code for:
+* the "project page", which shows a playable version of the project, along with the project's title, description, comments, remixes and studios; this page operates in the background when you "See inside" a project
+* the site's home page
+* the Ideas page
+* landing pages for various Scratch extensions, such as LEGO Mindstorms and Micro:bit
```

Comment:
Remember, the correct capitalization is LEGO MINDSTORMS and micro:bit!

Reply:
lol tHan:KS!